codice/ddf
# Ddf 1916- Integrate Search UI 'help' page into documentation

#### What does this PR do?
Integrates Search UI 'Help' end user documentation into complete documentation.

#### Who is reviewing it?
@jrnorth @mcalcote @Lambeaux @oscaritoro @vinamartin @stustison 
 
#### How should this be tested?
Successful build, no 'Unresolved Directive' error messages in compiled documentation, no broken image links (see below for exceptions)

#### Any background context you want to provide?
Some images are missing from the Help/Using section. See [DDF-2013](https://codice.atlassian.net/browse/DDF-2013)

#### What are the relevant tickets?
[DDF-1916](https://codice.atlassian.net/browse/DDF-1916) 
#### Screenshots (if appropriate)
#### Checklist:
